courses/free-courses-en.md

```diff
@@ -523,6 +523,7 @@
 * [Deep Learning With Tensorflow 2.0 and Keras](https://youtube.com/playlist?list=PLeo1K3hjS3uu7CxAacxVndI4bE_o3BDtO) - codebasics
 * [Deep Multi-Task and Meta Learning](https://cs330.stanford.edu) - Chelsea Finn (Stanford University)
 * [Deep Reinforcement Learning](http://rail.eecs.berkeley.edu/deeprlcourse/) - Sergey Levine
+* [DeepLearning - Mitesh Khapra, SKS Iyengar || IIT Ropar and Madras - NPTEL](https://youtu.be/aPfkYu_qiF4?si=9dUks3nu1pgf_KXP) (NPTEL)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
we don't allow shortened links.
escape `|`
when you've made changes, please add `(changes made)` to the title of your PR